src/test/java/com/iexec/core/configuration/ResultRepositoryConfigurationTest.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2020-2024 IEXEC BLOCKCHAIN TECH
```

Comment:
We will speak about this later, but you cannot copyright something which did not exist in the past
So for newly created files:
```suggestion
 * Copyright 2024 IEXEC BLOCKCHAIN TECH
```